maxim/logger/portkey/portkey.py

```
@@ -0,0 +1,184 @@
+        try:
+            trace = self._logger.trace({"id": final_trace_id})
+            gen_config = {
+                "id": str(uuid4()),
+                "model": model,
+                "provider": "portkey",
+                "name": generation_name,
+                "model_parameters": OpenAIUtils.get_model_params(**kwargs),
+                "messages": OpenAIUtils.parse_message_param(messages),
+            }
+            generation = trace.generation(gen_config)
+        except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)